cmd/boulder/main.go

```diff
@@ -22,11 +24,45 @@ import (
 	"github.com/letsencrypt/boulder/wfe"
 )
 
+type timedHandler struct {
+	f     func(w http.ResponseWriter, r *http.Request)
+	stats statsd.Statter
+}
+
+func HandlerTimer(handler http.Handler, stats statsd.Statter) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		cStart := time.Now()
+		// FIX: this somehow goes negative sometimes?
+		stats.Inc("HttpConnectionsOpen", 1, 1.0)
```

Comment:
Statsd being lossy is finnicky about this sort of usage. I'd recommend one of two other approaches when the problem initially looks like "Inc/Dec" is the answer:
1. Make two counters (HttpConnectionsOpen HttpConnectionsClosed) and manipulate/subtract them at visualization time
2. Use a gauge and track the metric internally -- this is the only way to get exact answers